doc/classes/Camera.xml

```diff
@@ -75,9 +75,10 @@
 			<return type="Vector3">
 			</return>
 			<argument index="0" name="screen_point" type="Vector2">
+			<argument index="1" name="z_depth" type="float" default="0">
```

Comment:
There should be a `</argument>` before this line. The docs XML can be generated/validated with `godot --doctool .` from your build folder.

Reply:
My bad! Should be sorted now.